api/api.proto

```diff
@@ -4106,6 +4109,16 @@ message CollectDiagsJob {
   repeated DiagsCollectionStatus statuses = 2;
 }
 
+message DefragJob {
+  double max_duration_hours = 1;
+  uint32 max_nodes_in_parallel = 2;
+  repeated string nodes_select = 3;
```

Comment:
"nodes_select" and "node_selector" sound similar - can you rename ?
also add comments to the fields ?
do we need the schedule_id field in the job instance itself ?

Reply:
Will change `nodes_select` to `select_nodes`; change `node_selector` to `node_label_selector`
Will add comments.
The `schedule_id` is needed here, because we want to keep track of on which nodes did the defrag schedule stop each time and resume from there next time. We will need to store this information under schedule data structure. 
Maybe the other way also works: put the job_id into schedule data structure, and do a EnumerateSelect.